popup.js

```diff
@@ -1,3 +1,11 @@
+// I'm feling lucky button
+
+function lucky(){
+    var lButton = document.getElementById("imfl");
+    lButton.addEventListener("click", function () {
+        window.open("https://www.google.com/doodles");
```

Comment:
Yeah , whenever the search query is empty ,this code will work for that only.
@PaliwalSparsh  I'm working on it .